grunt_tasks/server.js

```diff
@@ -44,7 +44,8 @@ module.exports = function (grunt) {
     grunt.config.merge({
         shell: {
             readServerConfig: {
-                command: 'env python config_parse.py girder/conf/girder.local.cfg',
+                command: 'env python config_parse.py ' +
+                         grunt.config.get('girderDir') + '/conf/girder.local.cfg',
```

Comment:
Oh, if that's the only place we use them, I can just move this stuff to the dev.js script that doesn't run in the package environment. We can think about how to remove this part altogether as well.